main.go

```diff
@@ -1752,7 +1753,17 @@ func setLocationGPX(device ios.DeviceEntry, gpxFilePath string) {
 }
 
 func startLocationSimulation(service *instruments.LocationSimulationService, lat string, lon string) {
-	err := service.StartSimulateLocation(lat, lon)
+	latitude, err := strconv.ParseFloat(lat, 64)
+	if err != nil {
+		exitIfError("location simulation failed to parse lat", err)
+	}
+
+	longitude, err := strconv.ParseFloat(lon, 64)
+	if err != nil {
+		exitIfError("location simulation failed to parse lon", err)
+	}
```

Comment:
the `if` check happens in `exitIfError` already
```suggestion
	latitude, err := strconv.ParseFloat(lat, 64)
	exitIfError("location simulation failed to parse lat", err)

	longitude, err := strconv.ParseFloat(lon, 64)
	exitIfError("location simulation failed to parse lon", err)
```